packages/components/src/components/post-tooltip/post-tooltip.tsx

```diff
@@ -23,12 +24,14 @@ const tooltipTargetAttribute = 'data-tooltip-target';
  * @returns
  */
 const globalInterestHandler = (e: PointerEvent | FocusEvent) => {
-  const target = e.target as HTMLElement;
-  if (!target || !('getAttribute' in target)) return;
-  const tooltipTarget = target.getAttribute(tooltipTargetAttribute);
+  const targetElement = (e.target as HTMLElement).closest(
```

Comment:
Using `e.currentTarget` is an option as well, but it will move the tooltip to the child element which is unwanted.